drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -386,15 +395,21 @@ class RasterizerSceneGLES3 : public RendererSceneRender {
 			float IBL_exposure_normalization;
 
 			uint32_t fog_enabled;
+			uint32_t fog_mode;
 			float fog_density;
 			float fog_height;
 			float fog_height_density;
 
+			float fog_depth_curve;
+			float pad;
+			float fog_depth_begin;
```

Comment:
```suggestion

			float fog_height_density;
			float fog_depth_curve;
			float pad;
			float fog_depth_begin;
```
These should be grouped into sets of 4 to make alignment issues easy to spot.